src/app/components/feed/FeedClusterPage.js

```diff
@@ -22,6 +21,17 @@ const StyledTwoColumnLayout = styled.div`
   }
 `;
 
+const Column = styled.div`
```

Comment:
This was the last use of Column, so I just moved it to this page, since the FeedClusterPage will be deprecated after INE is completed